astropy/utils/xml/writer.py

```diff
@@ -8,12 +8,6 @@
 import contextlib
 import textwrap
 
-try:
-    import bleach
-    HAS_BLEACH = True
-except ImportError:
-    HAS_BLEACH = False
```

Comment:
This actually breaks the tests, as something else appears to be importing `HAS_BLEACH`.